src/app/pages/ConsensusAccountDetailsPage/ConsensusAccountDetailsView.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react'
+import { Account } from '../../../oasis-nexus/api'
+import { useTranslation } from 'react-i18next'
+import { CardEmptyState } from '../../components/CardEmptyState'
+import { ConsensusAccountData } from '../../components/Account'
+
+type ConsensusAccountDetailsViewProps = {
+  account: Account | undefined
+  isError?: boolean | undefined
+  isLoading?: boolean | undefined
+  showLayer?: boolean
+  standalone?: boolean
+}
+
+export const ConsensusAccountDetailsView: FC<ConsensusAccountDetailsViewProps> = ({
+  account,
+  isError,
+  isLoading,
+  showLayer,
+  standalone,
+}) => {
+  const { t } = useTranslation()
+
+  if (isError) return <CardEmptyState label={t('account.cantLoadDetails')} />
+  if (!account) return null
```

Comment:
We should probably better handle this? There is no account with this address, or something like that?

Reply:
See https://github.com/oasisprotocol/explorer/pull/1422#discussion_r1607801383

Reply:
@buberdds any reason why we are returning `null` here? I would expect a better explanation for this edge case. Or does it really never happens?

Reply:
I think when the query returns undefined for the data, then it also sets either `isLoading` or `isError` to true.
So I don't think actually happens.

Reply:
In any case, I have updated this to still show the "can't load details" empty filler screen.